azure-pipelines.yml

```diff
@@ -174,27 +174,22 @@ stages:
           targetPath: 'analyzers\tests\SonarAnalyzer.UnitTest\bin'
 
       - powershell: |
-          . .\scripts\utils.ps1
-
-          CreateFolder("coverage")
-          CreateFolder("TestResults")
-
-          & OpenCover.Console.exe -target:"dotnet.exe" -targetargs:"test analyzers\tests\SonarAnalyzer.UnitTest\bin\$(BuildConfiguration)\$(FrameworkMoniker)\SonarAnalyzer.UnitTest.dll -l trx -l console;verbosity=detailed" -returntargetcode -filter:"+[SonarAnalyzer.CFG]* +[SonarAnalyzer]* +[SonarAnalyzer.CSharp]* +[SonarAnalyzer.VisualBasic]*" -output:coverage/$(FrameworkMoniker).xml -safemode:off -excludebyattribute:*.ExcludeFromCodeCoverageAttribute
-          Test-ExitCode "ERROR: Unit tests for $(FrameworkMoniker) FAILED."
+          cd analyzers
+          & dotnet test -f $(FrameworkMoniker) -c $(BuildConfiguration) -l trx /p:AltCover=true,AltCoverForce=true,AltCoverVisibleBranches=true,AltCoverAssemblyFilter='Moq|Humanizer|AltCover',AltCoverReport=coverage/coverage.xml
         displayName: '.Net UTs'
 
       - task: PublishPipelineArtifact@1
         condition: always()
         displayName: 'Save coverage files'
         inputs:
-          path: 'coverage'
+          path: 'analyzers/tests/SonarAnalyzer.UnitTest/coverage'
           artifact: $(CoverageArtifactName)
 
       - task: PublishPipelineArtifact@1
         condition: always()
         displayName: 'Save test results files'
         inputs:
-          path: 'TestResults'
+          path: 'analyzers/tests/SonarAnalyzer.UnitTest/TestResults'
           artifact: $(TestResultsArtifactName)
 
       - task: PublishTestResults@2
```

Comment:
Do we need the sonar.cs.opencover.reportsPaths to be set if we don't use opencover anymore?

Reply:
probably the report is in OpenCover format? just like coverlet?

Reply:
Yes, I had an offline chat with @costin-zaharia-sonarsource who helped me realise that in the name of the property "opencover" is not the name of the tool we use, but the format of the report.